docs/plugin-development.rst

```diff
@@ -57,7 +57,14 @@ The contents of plugin.json for our example will be:
 This information will appear in the web client administration console, and
 administrators will be able to enable and disable it there. Whenever plugins
 are enabled or disabled, a server restart is required in order for the
-change to take effect.
+change to take effect. 
+
+If you are developing a plugin for girder, sometimes using the Rebuild and restart button 
+on Plugins page may be undesirable as it will rebuild core and all enabled plugins 
+in production mode, which will take some time and doesn't provide sourcemap.
+Rebuild specific plugin restart the server may be a better choice. See `During Development <development.html#during-development>`__ for detail.
```

Comment:
@zachmullen I think I wasn't sure `watch` would be considered as automatic or manual when I wrote this. Do you see what I meant? Do you other words in mind? `specifically`?